packages/next/src/server/require-hook.ts

```diff
@@ -13,6 +13,14 @@ const hookPropertyMap = new Map()
 
 let aliasedPrebundledReact = false
 
+// that env var is only set in app router
+const isApp = !!process.env.__NEXT_PRIVATE_PREBUNDLED_REACT
+const currentRuntime = `${
+  isApp
+    ? 'next/dist/compiled/next-server/app-page.runtime'
+    : 'next/dist/compiled/next-server/pages.runtime'
+}.prod`
```

Comment:
Do we need to move this inside of the require handling in case the require hook is loaded before the env variable is updated?